README.md

```diff
@@ -62,16 +62,16 @@ Summary of new features:
 
 + A core language based on "Quantitative Type Theory" which allows explicit
   annotation of erased types, and linear types.
-+ `let` bindings are now more expressive, and can be used to define pattern
++ `let` bindings are now more expressive, and can be used to define pattern-
   matching functions locally.
-+ Names which are in scope in a type are also always in scope in the body of
++ Names that are in scope in a type are also always in scope in the body of
   the corresponding definition.
 + Better inference. Holes are global to a source file, rather than local to
   a definition, meaning that some holes can be left in function types to be
   inferred by the type checker. This also gives better inference for the types
-  of `case` expressions, and means fewer annotations are needed in interface
+  of `case` expressions and means fewer annotations are needed in interface
   declarations.
-+ Better type checker implementation which minimises the need for compile
++ Better type checker implementation which minimizes the need for compile-
```

Comment:
Not errors.

I'm not sure what we decided about British spelling or not.

The OED only contains 'compile time' as a draft addition noun, the adjective is completely missing.